packages/client-adapters/tests/client-adapters.test.js

```diff
@@ -672,4 +672,25 @@ describe('client-adapters.test.js', () => {
 
     console.info('resDoc.result - ', resDoc.result)
   }, 50000)
+
+  it('30 ClientAdapters - detectLangMethod - getDetectedLangsList - detects language from the text (sourceData)', async () => {
+    ClientAdapters.init()
+
+    let res1 = await ClientAdapters.detectlangGroup.detectlang({
+      method: 'getDetectedLangsList',
+      params: {
+        sourceData: { data: {
+          detections: [
+            { language: 'en', isReliable: true, confidence: 3.36 },
+            { language: 'pt', isReliable: false, confidence: 3.36 },
+            { language: 'eu', isReliable: false, confidence: 3.36 }

```

Comment:
I'd recommend rewriting this text so that two elements have `isReliable: true`, but with a different `confidence`. That way, it would test that sorting is working correctly.